lib/livebook/zta.ex

```diff
@@ -45,6 +45,12 @@ defmodule Livebook.ZTA do
   """
   @callback authenticate(name(), Plug.Conn.t(), keyword()) :: {Plug.Conn.t(), metadata() | nil}
 
+  @doc """
+  Logouts against the given name.
+  """
+  @callback logout(name(), Phoenix.LiveView.Socket.t()) :: :ok | {:error, String.t()}
+  @optional_callbacks logout: 2
+
```

Comment:
I think we should remove this because the goal is to extract this to a separate library and this functionality is specific to teams, which will not be part of the library. :)

```suggestion
```